.github/workflows/release.yml

```diff
@@ -46,7 +46,7 @@ jobs:
 
       - name: Update versions in installation docs
         run: |
-          LATEST_RELEASE_VERSION=$(curl -s -XGET  https://api.github.com/repos/kubeshop/botkube/releases/latest | awk -F'"tag_name": ' '{ print $2 }' | tr -d "\n" | awk -F'"' '{ print $2 }')
+          LATEST_RELEASE_VERSION="v${{ inputs.version }}.0"
```

Comment:
@huseyinbabal will it work with RC versions which has `-rc.x` suffix? 🤔 do we need the fixed `.0` suffix?

Reply:
Actually, the main motivation is, render the doc in advance. We didn't want to add rc versions in the doc, since we will need to do 2nd release  to add non-rc version once it is released. So let say that, you made release cut for 0.15.0, you have release branch ready release-0.5, and you have rc version 0.15.0-rc.1 in botkube repo. You can go to docs repo, and trigger the release with 0.15. It will create a release pr containing the version 0.15.0 which is fine

Reply:
Ah, okay 👍 Thank you for explanation, sound good 👍